recipes/slurm_config.rb

```diff
@@ -42,3 +42,11 @@
 link '/etc/profile.d/slurm.csh' do
   to '/opt/slurm/etc/slurm.csh'
 end
+
+# Ensure slurm user can sudo on Slurm commands
```

Comment:
Minor: it would be nice for the comment here to state that this isn't strictly necessary but rather that it is done for the sake of consistency (code used by both cluster management daemons and suspend/resume uses same path).